test/truefi2/lines-of-credit/TrueRateAdjuster.test.ts

```diff
@@ -72,6 +72,89 @@ describe('TrueRateAdjuster', () => {
     })
   })
 
+  describe('addPoolToTVL', () => {
```

Comment:
I noticed this *describe* and the next one are under `initialize` *describe*. Let's move them out of there.

Reply:
Nevermind, everything's good.

Reply:
Are you sure? It doesn't look like nested to me, and when I run the unit tests they don't show up under the `initializer` subtree.